crates/fluence-app-service/src/service.rs

```diff
@@ -172,7 +160,9 @@ impl AppService {
             module.config.extend_wasi_envs(envs.clone());
             module
                 .config
-                .extend_wasi_files(preopened_files.clone(), mapped_dirs.clone());
+                .extend_wasi_files(<_>::default(), mapped_dirs.clone());
+
+            module.config.set_service_base_dir(&service_dir)
```

Comment:
I'm not sure you need custom `mapped_dirs` if you call `set_service_base_dir` anyway 🤔

Reply:
Yep, changed it to preopened files